src/lib.rs

```diff
@@ -97,142 +97,142 @@ pub mod efi {
     //
 
     pub use crate::base::Boolean;
-    pub use crate::base::Char8;
     pub use crate::base::Char16;
+    pub use crate::base::Char8;
+    pub use crate::base::Event;
     pub use crate::base::Guid;
-    pub use crate::base::Status;
     pub use crate::base::Handle;
-    pub use crate::base::Event;
+    pub use crate::base::ImageEntryPoint;
     pub use crate::base::Lba;
-    pub use crate::base::Tpl;
     pub use crate::base::PhysicalAddress;
+    pub use crate::base::Status;
+    pub use crate::base::Tpl;
     pub use crate::base::VirtualAddress;
-    pub use crate::base::ImageEntryPoint;
```

Comment:
The ordering here is actually aligned with the spec. It makes it very easy to find things if you are reading the spec in parallel, or if you work with other UEFI language bindings.

This is not a huge deal, but is there a way to avoid rustfmt ordering these alphabetically? I will check it myself as well.

Reply:
I just merged this, it is not worth spending more time on, just to get a different ordering.